### PR TITLE
Remove `ansible`, update `requirements.txt`

### PR DESCRIPTION
**Pull request by portante**
## Changes introduced with this PR

Update `requirements.txt` based on the changes to `pyproject.toml`.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,6 @@ license = "Apache-2.0+GPL-2.0-only"
 [tool.poetry.dependencies]
 python = "^3.9"
 arcaflow-plugin-sdk = "0.9.0"
-ansible = "^6.5.0"
```
Comment (dustinblack) [inline pyproject.toml:11]: The ansible module is still used in `metadata_plugin.py`. Please revert this change.

**Review by dustinblack — CHANGES_REQUESTED** [review]

**Review by dustinblack — APPROVED** [review]

**Review by portante — COMMENTED** [review]
This now has the updated modules.
